index.js

```diff
@@ -51,6 +51,12 @@ const defaults = {
  * can open. This is similar to `options.allowWarmUp` in that no matter how many
  * failures there are, if the number of requests within the statistical window
  * does not exceed this threshold, the circuit will remain closed. Default: 0
+ * @param options.errorFilter {Function} an optional function that will be
+ * called when the circuit 's function fails (returns a rejected Promise). If
```

Comment:
Is the extra space before each `'s` in this paragraph intentional? 
```suggestion
 * called when the circuit's function fails (returns a rejected Promise). If
```

Reply:
Good catch. My IDE did some strange reformatting when I copied/pasted. Thought I had caught it all.